packages/firebase_admob/README.md

```diff
@@ -182,6 +182,95 @@ RewardedVideoAd.instance.listener =
 Because `RewardedVideoAd` is a singleton object, it does not offer a `dispose`
 method.
 
+## Using native ads
+
+Native ads are presented to users via UI components that
+are native to the platform. (e.g. A
+[View](https://developer.android.com/reference/android/view/View) on Android or a
+[UIView](https://developer.apple.com/documentation/uikit/uiview?language=objc)
+on iOS). Using Flutter widgets to create native ads is NOT supported by
+this.
+
+### Android
+The Android Admob Plugin requires a class that implements `NativeAdFactory` which calls `createNativeAd(
+[UnifiedNativeAd](https://developers.google.com/android/reference/com/google/android/gms/ads/formats/UnifiedNativeAd) nativeAd,
+Map<String, Options> customOptions)` and returns a
+[UnifiedNativeAdView](https://developers.google.com/android/reference/com/google/android/gms/ads/formats/UnifiedNativeAdView).
+
+You can implement this in your `MainActivity.java` or create a separate class in the same directory
```

Comment:
Do we want to recommend doing this in the activity?

Reply:
The example of implementing a `UnifiedNativeAdView` requires a `LayoutInflater`, which you would get from an `Activity`. I assumed it would be ok to recommend the path of least resistance.